keyboards/gboards/gergoplex/rules.mk

```diff
@@ -0,0 +1,14 @@
+MCU = atmega32u4
+
+BOOTLOADER = atmel-dfu
+
+CUSTOM_MATRIX    = yes
```

Comment:
Now when I have got the information about matrix pinout for AVR-controller keyboard half, I intend to try `CUSTOM_MATRIX = lite`, and try my chances with `SPLIT_KEYBOARD = true`.

Reply:
`SPLIT_KEYBOARD = yes` is meant solely for dual controller keyboards.  So it won't work for any of germ's boards. 

However, you should check out the ergodox ez, now, as the board has been converted to custom matrix lite.

Reply:
Thanks for the hint!